packages/snaps-cli/src/commands/manifest/implementation.ts

```diff
@@ -1,11 +1,30 @@
 import { checkManifest, indent } from '@metamask/snaps-utils/node';
 import { assert } from '@metamask/utils';
 import { red, yellow, green } from 'chalk';
+import { promises as fs } from 'fs';
 import type { Ora } from 'ora';
 import { dirname } from 'path';
+import babel from 'prettier/parser-babel';
+import { format } from 'prettier/standalone';
 
 import { error, info, warn } from '../../utils';
 
+/**
+ * Write the manifest to disk.
+ *
+ * @param path - The path to write the manifest to.
+ * @param data - The manifest data.
+ * @returns A promise that resolves when the manifest has been written.
+ */
+async function writeManifest(path: string, data: string) {
```

Comment:
Should this be optional?

Reply:
Should formatting with Prettier be optional?

What configuration does it use? Can a dev overwrite it?

Reply:
> Should formatting with Prettier be optional?

It could be, but I don't see why it should be. Right now it's formatted with whatever `JSON.stringify` uses, so it doesn't make much of a difference really.

> What configuration does it use? Can a dev overwrite it?

It uses the default configuration, and can't be overridden. I can add config detection if you prefer.

Reply:
Let's at least let devs override the config with their own yeah, otherwise use the default